Framework/PythonInterface/plugins/algorithms/SeparateMonitorSpectra.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import (absolute_import, division, print_function)
+from mantid.simpleapi import *
+from mantid.kernel import *
+from mantid.api import (MatrixWorkspaceProperty, DataProcessorAlgorithm, PropertyMode)
+
+class SeparateMonitorSpectra(DataProcessorAlgorithm):
```

Comment:
This sounds like a useful addition but I think a name such as `ExtractMonitors` or `ExtractMonitorSpectra` (my preference would be for the shorter first one) would be better. We tend to use the work `extract` over `separate`.